### DISCUSSION TITLE
Add OrgId to the Dashboard and Panel URL created by the notifier

### DISCUSSION THREAD

**Discussion by Vimao** [grafana/grafana#44578] [Alerting]
We have multiple internal organizations in grafana in order to control what the users can see and interact with. Those organizations have dashboards with the same UID. We also have one team that has access to all the organizations for the purpose of monitoring.

The team that has access to everthing is using OpsGenie to operate on the alerts. 

When Grafana sends the notification, it should send the link with the parameter ?orgid=x with the value of the orgid for which the alert is defined. Otherwise the operator has to constantly switch between orgs manually, which sometimes leads to confusion when viewing dashboards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
